Tasks/NuGetCommandV2/nugetrestore.ts

```diff
@@ -2,18 +2,16 @@ import * as path from "path";
 import * as tl from "vsts-task-lib/task";
 import {IExecOptions, IExecSyncResult} from "vsts-task-lib/toolrunner";
 
-import * as auth from "nuget-task-common/Authentication";
-import { IPackageSource } from "nuget-task-common/Authentication";
-import * as commandHelper from "nuget-task-common/CommandHelper";
-import locationHelpers = require("nuget-task-common/LocationHelpers");
-import {NuGetConfigHelper2} from "nuget-task-common/NuGetConfigHelper2";
-import * as ngToolRunner from "nuget-task-common/NuGetToolRunner2";
-import peParser = require("nuget-task-common/pe-parser/index");
-import {VersionInfo} from "nuget-task-common/pe-parser/VersionResource";
-import * as nutil from "nuget-task-common/Utility";
-import * as pkgLocationUtils from "utility-common/packaging/locationUtilities";
+import * as auth from "packaging-common/nuget/Authentication";
+import * as commandHelper from "packaging-common/nuget/CommandHelper";
+import {NuGetConfigHelper2} from "packaging-common/nuget/NuGetConfigHelper2";
+import * as ngToolRunner from "packaging-common/nuget/NuGetToolRunner2";
+import peParser = require("packaging-common/pe-parser/index");
+import {VersionInfo} from "packaging-common/pe-parser/VersionResource";
+import * as nutil from "packaging-common/nuget/Utility";
+import * as pkgLocationUtils from "packaging-common/locationUtilities";
 import * as telemetry from "utility-common/telemetry";
-import INuGetCommandOptions from "./Common/INuGetCommandOptions";
+import INuGetCommandOptions from "packaging-common/nuget/INuGetCommandOptions2";
```

Comment:
interesting... was this just an old mistake, this command was using the old tool runner environment settings?

Reply:
No, it was using the new ones, but it defined them in its own file for some reason.